frontend/scripts/react-components/chart/tick/tick.component.jsx

```diff
@@ -10,12 +11,21 @@ function Tick(props) {
   return (
     <g transform={`translate(${x},${y})`}>
       <defs>
-        <filter x="0" y="0" width="1" height="1" id="solid">
-          <feFlood floodColor={backgroundColor || '#fff'} />
-          <feComposite in="SourceGraphic" />
-        </filter>
+        {!isIe() && (
```

Comment:
filters will cause issues on the ticks, as in IE it will render them as black blocks, we simply ignore rendering it on IE and it will show